src/routes/indexing-route.php

```diff
@@ -267,8 +267,6 @@ public function __construct(
 		$this->indexable_indexing_complete_action  = $indexable_indexing_complete_action;
 		$this->indexing_complete_action            = $indexing_complete_action;
 		$this->prepare_indexing_action             = $prepare_indexing_action;
-		$this->post_link_indexing_action           = $post_link_indexing_action;
-		$this->term_link_indexing_action           = $term_link_indexing_action;
```

Comment:
Since Lars is not working today, and this PR has some priority, I am implementing these changes myself 🙂 .

I added this to the _Relevant technical choices_ section:
> Removed duplicated post_link_indexing_action and term_link_indexing_action property assignments in the Indexing_Route constructor.